### ISSUE TITLE
JAWS does not announce aria interactable attributes (such as expanded, pressed, etc) when page is inside an iframe that is inside a webview2 in a win32 application

### ISSUE THREAD

**Issue by ros3cin** [FreedomScientific/standards-support#818]
### Summary

Issue happens for any page inside an iframe inside a webview2 of a win32 app.

1. Open the attached file and extract its contents [JAWS not announcing win32 inside iframe inside webview.zip](https://github.com/FreedomScientific/standards-support/files/14780706/JAWS.not.announcing.win32.inside.iframe.inside.webview.zip)
2. Open the executable - that will open a jsfiddle page with a repro code
3. Run the jsfiddle
4. Turn on JAWS
5. Use tab to navigate to the toggle button
6. Press space/enter to activate the toggle button

### Expected result

JAWS should announce expanded or collapsed.

### Actual result

JAWS doesn't announce expanded or collapsed

### Additional Information
Scenario works correctly with NVDA and Windows Narrator

#### JAWS version and build number
JAWS professional edition
Version 2024.2403.3

#### Operating System and version
OS Name	Microsoft Windows 11 Enterprise
Version	10.0.22631 Build 22631

#### Browser and version:
Edge inside webview2
